src/plugins/expressions/public/plugin.ts

```diff
@@ -34,6 +34,9 @@ export interface ExpressionsStart extends ExpressionsServiceStart {
   ReactExpressionRenderer: typeof ReactExpressionRenderer;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ExpressionsStartDependencies {}
```

Comment:
It doesn't seem like it's been used anywhere. Is this intentional?

Reply:
Good point, thanks!